manuscript/01.introduction.md

```diff
@@ -4,15 +4,14 @@
 
 <!-- tHMMs are a solution to modeling lineage data -->
 Hidden Markov models (HMMs) provide an efficient strategy to infer discrete states from measurements when a series of co-dependent observations are made. An example of this is their wide-spread use in time series analysis, where each measurement is dependent on those that came before [@PMID:15629048; @DOI:10.1109/5.18626]. Recognizing this co-dependence allows HMMs to make accurate inferences even in the presence of extremely noisy measurements since each neighboring measurement provides accumulating evidence for a prediction [@DOI:10.1126/scisignal.aao1917]. HMMs have been adapted to lineage trees (tHMMs) so that each measurement across the tree can similarly provide accumulating evidence for a prediction. Just like with time-series data, these models can provide very accurate predictions despite noisy measurements and limited information by recognizing the co-dependence between related measurements [@doi:10.1109/78.668544; @doi:10.1109/TSP.2004.832006]. tHMMs have been used in a multitude of applications, from image classification to comparative genomics [@pmid:18255546; @pmid:23762278]. In cells, these models have been fit to lineages collected from stem cells and bacteria colonies but have always required tailor-made implementations [@PMID:19628503; @doi:10.1101/488981]. Improvements in cell tracking and high-throughput imaging promise to make these models valuable techniques for studying the plasticity of heterogeneous cell populations. However, widespread use will require usable implementations that can be readily adapted to different experimental measurements, examples of unique insights revealed, and standards for experimental validation.
```

Comment:
In this sentence `However, widespread use will require usable implementations that can be readily adapted to different experimental measurements, examples of unique insights revealed, and standards for experimental validation.` I'm a bit confused on the placement of the phrase 'examples of unique insights revealed.' Is it a part of a list of things that widespread will require or is it meaning that the unique insights showed that the implementations need to be adapted to different experimental measurements?

Reply:
Other than this, the edits sounded clear to me and I did not notice any typos!

Reply:
What I'm trying to say is examples of tHMM models working to discover new things will motivate people to use them. Do you have another way or phrasing this? I agree it's awkward.

Reply:
`However, widespread use will require usable implementations that can be readily adapted to different experimental measurements, examples of successful tHMM applications that will encourage increased usage, and standards for experimental validation.`

Is that the message you are trying to convey? It might be lengthy sentence, however.

Reply:
@jclagarde this is excellent. Please replace the sentence. Thanks!